components/Accordion/Accordion.styles.ts

```diff
@@ -15,14 +15,14 @@ export const collapseAllIcon = '-mt-02em';
 
 // Accordion item
 export const list = 'list-unstyled rs-mt-1';
-export const listItem = (color: AccordionColorType) => cnb('relative mb-0 border-b first:border-t', accordionColors[color || 'palo-alto-light']);
+export const listItem = (isDigitalRed: boolean) => cnb('relative mb-0 border-b first:border-t', accordionColors[isDigitalRed ? 'digital-red' : 'palo-alto-light']);
```

Comment:
The boolean check `isDigitalRed` passed in has taken care of the case where no color is selected in Storyblok (user can still manually clear a selection even if the "hide empty option" box is checked).